bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -291,6 +291,13 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
         } yield new Problem(Severity.Warning, s"The R600 is little used and may be difficult to schedule.", "Observations", s.inObsListView(o.band, _.Fixes.fixBlueprint(b)))
     }
 
+    private val f2MOSCheck = for {
+      o <- p.nonEmptyObservations
+      b <- o.blueprint
+      if b.isInstanceOf[Flamingos2BlueprintMos]
```

Comment:
I knew there had to be a better way to do this, but I couldn't think of it on the fly, so I just followed suit with the other `val`s and `def`s here and used the ugly `isInstanceOf`.

Since Travis probably won't finish this before I leave for the day, I'll go ahead and make this change!